docs/rules/prefer-array-some.md

```diff
@@ -0,0 +1,41 @@
+# Prefer `.some(…)` over `.find(…)`.
+
+Prefer use [`Array#some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) over [`Array#find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) when testing array.
```

Comment:
"when testing array" is not clear enough about what it does.

Reply:
But  we don't know what `.find` is doing.

Reply:
Something like:

> when ensuring at least one element in the array passes a given check.